src/components/base/ErrorMessage/ErrorMessage.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import React from 'react'
+import classNames from 'classnames'
+
+import ErrorMessageStyle from './ErrorMessage.m.scss'
+
+type Theme = 'info' | 'warning' | 'error'
+type Props = StyleComponent<Theme> & {
+  message: string,
+}
+
+function ErrorMessage(
+  {
+    theme = 'error',
+    message = '',
+    className,
+  }: Props) {
+  return (
+    <span className={classNames(
+      className,
+      ErrorMessageStyle.core,
+      ErrorMessageStyle[theme])}
+    >
+      {message}
+    </span>
+  )
+}
+
+const MemorizedErrorMessage = React.memo/* :: <Props> */(ErrorMessage)
```

Comment:
`Memoized` is correct [term](https://en.wikipedia.org/wiki/Memoization#Etymology)